### DISCUSSION TITLE
how to disable quick app running by first letter type?

### DISCUSSION THREAD

**Discussion by sanakess** [Open-Shell/Open-Shell-Menu#775] [Q&A]
Hi, when i turned off  "show search box" in basic settings, i got the opportunity to launch applications by press first letter of program in the list. How to disable this? I type "s" and "ssh client"  starts. I try to troubleshoot bug: search in "access normaly"  mode in basic settings does not always work, and instead of searching for something i running application

**Comment by hazymat** [comment]
+1 for this question

**Comment by Duke-M-commits** [comment]
#447
